### ISSUE TITLE
window.require is broken after including Handsontable

### ISSUE THREAD

**Issue by dchekanov** [handsontable/handsontable#2554]
Hi,

I use RequireJS in my app. After requiring Handsontable plugin (0.15.0-beta6), RequireJS stops functioning.
It looks like Handsontable overwrites window.require with it's own version to handle internal modules.
When required module is not found among internal ones, it calls previous version of window.require like this:

```javascript
if (previousRequire) return previousRequire(name, true);
```

Which doesn't work since the original window.require works with different parameters:

```javascript
function (deps, callback, errback, optional)
```

I guess it's a typo, and the correct code should look something like this:

```javascript
if (previousRequire) return previousRequire.apply(this, arguments);
```

**Comment by budnix** [comment]
Fixed in our build tool https://github.com/handsontable/hot-builder/commit/171aca2d893aedea79f10f95888932825d1a530b. It will be released tomorrow with Handsontable 0.15.0.

**Comment by dchekanov** [comment]
I've updated to 0.15.0 and it works fine with RequireJS so far, thank you.
